homeassistant/components/modbus/switch.py

```diff
@@ -179,10 +186,9 @@ def available(self) -> bool:
         self._available = True
 
 
-class ModbusRegisterSwitch(ModbusCoilSwitch):
+class ModbusRegisterSwitch(ModbusSwitch):
     """Representation of a Modbus register switch."""
 
-    # pylint: disable=super-init-not-called
```

Comment:
Why not simply call super(), in case there is a reason not to call super() a comment would be nice.

Reply:
Pylint is probably a leftover. I didn't disable it by purpose. I will take a look at it.

Reply:
Pylint is a leftover from original code. I actually removed that pylint line, because now I call `super()`.